run/order.go

```diff
@@ -145,22 +146,50 @@ func BuildDAG(
 
 	visited[s.Path()] = struct{}{}
 
-	logger.Trace().
-		Msg("Load all stacks in dir after current stack.")
-	afterStacks, err := loader.LoadAll(root, s.HostPath(), s.After()...)
+	cleanOrderPaths := func(fieldname string, paths []string) []string {
+		cleanpaths := []string{}
+		for _, path := range paths {
+			var abspath string
+			if filepath.IsAbs(path) {
+				abspath = filepath.Join(root, path)
+			} else {
+				abspath = filepath.Join(s.HostPath(), path)
+			}
+			st, err := os.Stat(abspath)
+			if err != nil {
+				logger.Warn().
+					Err(err).
+					Msgf("failed to stat %q path %q - ignoring", fieldname, abspath)
+			} else if !st.IsDir() {
+				logger.Warn().
+					Msgf("stack.%s path %q is not a directory - ignoring",
+						fieldname, path)
+			} else {
+				cleanpaths = append(cleanpaths, path)
+			}
+		}
+		return cleanpaths
+	}
+
+	afterPaths := cleanOrderPaths("after", s.After())
+	beforePaths := cleanOrderPaths("before", s.Before())
+
+	logger.Trace().Msg("Load all stacks in dir after current stack.")
```

Comment:
```suggestion
	logger.Trace().Msg("load all stacks in dir after current stack")
```

I think the final dot on the log messages don't help much, lately I have been avoiding them.

Reply:
but logs are always capitalized, no?